community/modules/database/slurm-cloudsql-federation/variables.tf

```diff
@@ -97,6 +131,11 @@ variable "private_vpc_connection_peering" {
   default     = null
 }
 
+variable "subnetwork_self_link" {
+  description = "Self link of the network where Cloud SQL instance PSC endpoint will be created"
+  type        = string
```

Comment:
Please specify `default = null`, after saw recent problems with terraform google providers (version>6) when "" is used as "empty" value for `subnetwork_project `.
I'm not sure that this concern is applicable in this situation.